firmware/host/include/occli_common.h

```diff
@@ -74,43 +74,43 @@ typedef struct {
     char actionType[OCMW_MAX_ACTION_SIZE];
     char arguments[OCMW_HELP_FRAME_SIZE];
     char parameter[OCMW_HELP_FRAME_SIZE];
-}helpMenu;
+} helpMenu;
 
 typedef struct {
     char subsystem[OCCLI_CHAR_ARRAY_SIZE];
     char component[OCCLI_CHAR_ARRAY_SIZE];
     char subComponent[OCCLI_CHAR_ARRAY_SIZE];
-}commandFrame;
-/*
+} commandFrame;
 
+/*
  * Initialize the ocmw communication
  */
-int32_t occli_init_comm(void);
+extern int32_t occli_init_comm(void);
 /*
  * Deinitialize the ocmw communication
  */
-int8_t occli_deinit_comm(void);
+extern int8_t occli_deinit_comm(void);
 /*
  * @param cmd an input string (by pointer)
  * @param cmdlen an input value (by value)
  *
  * @return true if function succeeds, false otherwise
  */
-int32_t occli_send_cmd_to_ocmw(const char *cmd, int32_t cmdlen);
+extern int32_t occli_send_cmd_to_ocmw(const char *cmd, int32_t cmdlen);
```

Comment:
Are we need to make all function extern? Will it still give error if we only make inline to static ?

Reply:
The compiler doesn't need extern on prototypes but it's good practice for readability.

Reply:
Are you planning to change all other headers file as well into host?

Reply:
No, only these changes were necessary.

Reply:
I think it would be nice if all the files(.h) have extern added which will give a uniform appearance in the code.

Reply:
Added externs to other include files.  I think normally a change that involves files that are not part of the issue, should be done in a separate issue.  But in this case it was straightforward.  Most importantly, adding the extern in the second commit didn't change the md5sum of the build objects.